src/singletons/StreamerMode.cpp

```diff
@@ -194,9 +202,15 @@ bool StreamerMode::isEnabled() const
     return this->private_->isEnabled();
 }
 
+void StreamerMode::start()
+{
+    this->private_->start();
+}
+
 StreamerModePrivate::StreamerModePrivate(StreamerMode *parent)
     : parent_(parent)
 {
+    this->thread_.setObjectName("StreamerMode");
```

Comment:
This could use `renameThread`. Would be nice to only use a single mechanism to name threads